### PR TITLE
arm: aspeed: Update secure memory layout

### PR DESCRIPTION
**Pull request by ChiaweiW**
Most Aspeed AST2600/AST2700 EVBs are equipped with 1GB DRAM.
This patch updates the secure memory layout (TZDRAM) accordingly.

### CODE REVIEW

File: core/arch/arm/plat-aspeed/conf.mk [hunk 1]
```diff
@@ -34,10 +34,10 @@ $(call force,CFG_LPAE_ADDR_SPACE_BITS,36)
 CFG_NUM_THREADS ?= $(CFG_TEE_CORE_NB_CORE)
 
 CFG_DRAM_BASE ?= 0x400000000
-CFG_DRAM_SIZE ?= 0x400000000
+CFG_DRAM_SIZE ?= 0x40000000
```
Comment (jenswi-linaro) [inline core/arch/arm/plat-aspeed/conf.mk:37]: Unintended change?

Comment (jenswi-linaro) [reply core/arch/arm/plat-aspeed/conf.mk:37]: OK, thanks.

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by ChiaweiW** [comment]
Tag collected. Thanks!

**Pull request merged**